packages/teleport/src/Roles/templates/role.yaml

```diff
@@ -31,15 +31,14 @@ spec:
      - '*'
     # List of database users allowed to open database connections with
     db_users:
-    - dba-admin
+    - developer
     
     # List of windows desktop access labels that users can open desktop sessions to
     windows_desktop_labels:
       '*': '*'
     # Windows logins a user is allowed to use for desktop sessions.
     windows_desktop_logins:
-    - '{{internal.windows_logins}}'
```

Comment:
I would keep {{internal}} it's a good time to educe the user on what it is.

Reply:
done for all applicable